src/views/Home.jsx

```diff
@@ -1,11 +1,59 @@
+import { generateToken } from '@the-collab-lab/shopping-list-utils';
+import Button from '../components/Button';
 import './Home.css';
+import { useState } from 'react';
+// import { createCollection } from '../api/firebase';
+
+export function Home({ setListToken }) {
+	const [retrievedToken, setRetrievedToken] = useState('');
+
+	/**
+	 * This function generates a new unique Token corresponding to a new shopping list.
+	 */
+	const createNewList = () => {
+		const newToken = generateToken();
+		setListToken(newToken);
+
+		/**
+		 * We were creating a new collection in the database using the newToken:
+		 * createCollection(newToken);
+		 */
+	};
+
+	/**
+	 *  This function ensures that the retrievedToken state always reflects the current value in the input field.
+	 */
```

Comment:
I'm glad to see that you're excited to add features, but `handleTokenInput` and `joinExistentList` are out of this issues current scope and will be handled in [#5](https://github.com/the-collab-lab/tcl-63-smart-shopping-list/issues/5)

Reply:
Thank you for your feedback @mentalcaries! Should we delete those functions or comment them out?